networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "axo",
+    "decimals": 6,
```

Comment:
This token has 9 decimals, not 6: https://cexplorer.io/asset/asset16gtljf3uzghxyml9cjh9uvwg5jmcz25gy6kelu

Reply:
Corrected.